Automattic/jetpack
# Enhancement: Updating plans page illustrations

Currently on the plans page within the Jetpack plugin, we're displaying out of date plans illustrations. This PR seeks to update those illustrations with our new plans illustrations.

To Test:
1) Sync with this branch
2) Visit the plans tab while on the jetpack dashboard
Optional: switch your plan level to see the different plans illustrations

**Example of current illustration for personal plan (1 of 3 shown)**
<img width="761" alt="screen shot 2017-08-25 at 4 51 30 pm" src="https://user-images.githubusercontent.com/214813/29733372-ce0a76d8-89ba-11e7-8f05-5861b971b86a.png">


**Example of new illustration for personal plan**
<img width="778" alt="screen shot 2017-08-25 at 5 22 39 pm" src="https://user-images.githubusercontent.com/214813/29733386-df30ce44-89ba-11e7-87cc-64484bc4ef09.png">

<img width="801" alt="screen shot 2017-08-29 at 12 16 48 pm" src="https://user-images.githubusercontent.com/214813/29831719-00636422-8cb4-11e7-949d-3499238c9da4.png">

<img width="772" alt="screen shot 2017-08-29 at 12 18 57 pm" src="https://user-images.githubusercontent.com/214813/29831805-44dc2f9e-8cb4-11e7-9424-e441f095bfae.png">


**Illustration fixes on mobile screens (medium and small mobile screens)**
<img width="514" alt="screen shot 2017-08-29 at 12 25 55 pm" src="https://user-images.githubusercontent.com/214813/29832406-1e5ecc1c-8cb6-11e7-9eff-dffc3eccf2d6.png">
<img width="398" alt="screen shot 2017-08-29 at 12 31 12 pm" src="https://user-images.githubusercontent.com/214813/29832405-1e585ab2-8cb6-11e7-8597-6d6368e1f8ba.png">
